ec2_notify.py

```diff
@@ -74,20 +74,20 @@ def run(self):
 
         if 'public_ip' in ec2_info:
             message = message + ("<b> Resource Connection Info </b> <br />"
-                                 "Access Linux: "
+                                 "If you provisioned an EC2 instance: "
```

Comment:
should this mention linux at all?

Reply:
also this line will run kinda long so i would suggest adding a line break (</br />) at the end.